src/ensembl/src/globalActions.ts

```diff
@@ -1,15 +1,14 @@
 import { createAction } from 'typesafe-actions';
 import { BreakpointWidth } from './globalConfig';
+import { getGlobalAnalyticsObject } from './analyticsHelper';
```

Comment:
Would it be better to import `AnalyticsCategory` along with  `buildAnalyticsObject` and let the user choose the category they want to pass instead of having multiple exports for each individual category?

Reply:
I was thinking of making it easier for the user to build an analytics object. Do you think this is a bad idea?

Reply:
Also `AnalyticsCategory and `buildAnalyticsObject` are exported anyway. So the user has the choice to build it all by himself rather than using these specific functions.

Reply:
> So the user has the choice to build it using them rather than using these specific functions.

Yes, that would be my argument as well.